remotespark/wrapperkernel/sparkkernelbase.py

```diff
@@ -224,4 +238,4 @@ def _repeat_fatal_error(self):
         error = conf.fatal_error_suggestion().format(self._fatal_error)
         self._logger.error(error)
         self._ipython_display.send_error(error)
-        raise ValueError(self._fatal_error)
+        return self._complete_cell()
```

Comment:
It's a good idea but I don't personally know what this method returns, since we're offshoring all the work in this method to the IPython kernel.  Might require some investigation to see exactly what we should expect in order to add unit tests.